lib/ex_ice/ice_agent/impl.ex

```diff
@@ -198,9 +198,6 @@ defmodule ExICE.ICEAgent.Impl do
       notify(ice_agent.on_new_candidate, {:new_candidate, Candidate.marshal(cand)})
     end
 
-    # TODO should we override?
-    ice_agent = %{ice_agent | local_cands: ice_agent.local_cands ++ host_candidates}
```

Comment:
As gather_candidates can only be called after an ICE restart or in a :new state, local_cands will always be an empty list